src/GameOptions/ui/NumericDisplayOptions.tsx

```diff
@@ -4,10 +4,15 @@ import { Settings } from "../../Settings/Settings";
 import { OptionSwitch } from "../../ui/React/OptionSwitch";
 import { GameOptionsPage } from "./GameOptionsPage";
 import { FormatsNeedToChange } from "../../ui/formatNumber";
+import { OptionsSlider } from "./OptionsSlider";
 
 export const NumericDisplayPage = (): React.ReactElement => {
   const [locale, setLocale] = useState(Settings.Locale);
 
+  function handleFractionalDigitChange(_event: Event | React.SyntheticEvent, newValue: number | number[]): void {
+    Settings.fractionalDigits = newValue as number;
+    FormatsNeedToChange.emit();
+  }
```

Comment:
```suggestion
  }
  
```
Style: Newline between functions.